wikidict/__main__.py

```diff
@@ -23,7 +23,7 @@
                                 - "data/$LOCALE/dicthtml-$LOCALE.zip": Kobo format.
                                 - "data/$LOCALE/dict-$LOCALE.df": DictFile format.
   --find-templates          DEBUG: Find all templates in use.
-  --check-random-words N    Get and render N words.
+  --check-random-words=N    Get and render N words.
```

Comment:
Shoud we update the line 12 also?

Reply:
both are working, but sure, for consistency sake I will update it.